app/scripts/services/api-explorer.js

```diff
@@ -55,6 +55,27 @@
                // } else {
                //     return url;
                // }
+            },
+            createMethodList : function(_type, _url) {
```

Comment:
Please add some documentation for this new service.  What does it do, who calls it?